docs/simplecontroller_config.md

```diff
@@ -0,0 +1,39 @@
+# SimpleControllerConfiguration
+
+## Rate
+
+*float64*
```

Comment:
I've actually used JSON types in my other docs, so this would be just `*number*`. I am not in favor of anything, just thought this was about the JSON config. As long as we keep consistency I am okay with either approach and will update the rest.

Reply:
Agreed on consistency, and since we are documenting the configuration I am 50/50 on either side. One is correct if we consider the config file itself, the other one also can be considered correct since it is defining the *value* itself. @agnivade your thoughts?

Reply:
I think float64 is better since that matches with config.go

Reply:
Fair enough. We'll go with Golang's type system then.